spec/features/permissions/create_update_delete_permissions_spec.rb

```diff
@@ -188,10 +188,15 @@
             within '#collectionsChooser' do
               # selecting each individually as it seems more robust.
               select(@entry_id_1, from: 'Available Collections')
+              page.find('.from-container').native.send_keys :escape # press esc to hide tool tip, it blocks button
               find('button[title=add]').click
+              
               select(@entry_id_2, from: 'Available Collections')
+              page.find('.from-container').native.send_keys :escape # press esc to hide tool tip, it blocks button
```

Comment:
since all 3 instances of using escape here are so close together, maybe just the comment for the first time would be sufficient?

Reply:
will do